com.ibm.streamsx.json/com.ibm.streamsx.json/native.function/function.xml

```diff
@@ -29,6 +29,192 @@ Convert a value to JSON object with a single key encoded as a serialized JSON st
 </function:description>
         <function:prototype>&lt;string S, any T> public rstring toJSON(S key, T value)</function:prototype>
       </function:function>
+      <function:function>
+        <function:description>
+Extract values from JSON string accordingly to a given tuple.
+@param jsonString The input JSON string.
+@param tuple A mutable tuple to save extracted values.
+@return Reference to the input tuple.
+</function:description>
+        <function:prototype>&lt;tuple T> public T extractFromJSON(rstring jsonString, T value)</function:prototype>
+      </function:function>
+      <function:function>
+        <function:description>
+Parse JSON string (used in conjunction with queryJSON function).
```

Comment:
Please add two or three lines here explaining the role of the jsonIndex as handle. And the restrictions on usage in PEs and threads (maybe copied from the sample...)